ecl/compute/v2/servers/results.go

```diff
@@ -214,6 +214,9 @@ type Server struct {
 
 	// Fault contains failure information about a server.
 	Fault Fault `json:"fault"`
+
+	// ConfigDrive is the name of the server's config drive.
+	ConfigDrive string `json:"config_drive"`
```

Comment:
API Doc shows that type of config_drive is boolean/UUID(2). Seems like it is inconsistent.
https://ecl.ntt.com/en/documents/api-references/virtual-server/rsts/APIList/Virtual_Server_Compute/Compute/Create_server.html#request-parameter

Reply:
This structure stores the result of Get API, so I think we should refer to the API Doc below.
According to this API Doc, the data type of config_drive is String.
https://ecl.ntt.com/en/documents/api-references/virtual-server/rsts/APIList/Virtual_Server_Compute/Compute/Get_server_details.html#response-parameter

Reply:
Thanks. I resolved.